functions/src/world.ts

```diff
@@ -71,7 +71,6 @@ export const updateWorld = functions.https.onCall(async (data, context) => {
   }
 
   await checkIsWorldOwner(worldId, context.auth?.token.user_id);
```

Comment:
This should be more like `throwErrorIfNotWorldOwner({ worldId, userId });`

Reply:
Was fixed in https://github.com/sparkletown/sparkle/pull/2820/commits/aeb889a3ddc83638a1abdc657bc571aec9bcb95d